src/sentry/replays/endpoints/organization_replay_selector_index.py

```diff
@@ -59,7 +63,13 @@ def get_replay_filter_params(self, request, organization):
         return filter_params
 
     @handled_snuba_exceptions
+    @extend_schema(
+        operation_id="List an Organization's Selectors",
+        parameters=[GlobalParams.ORG_SLUG, ReplaySelectorValidator],
+        examples=ReplayExamples.GET_SELECTORS,
```

Comment:
I think this is missing the cursorqueryparam, and also the response body. Looks like there's no typeddict response body too 😢

Reply:
can we ensure that the docs test fails if there's no response body schema?

Reply:
It was failing in the openapi-diff / build api action was failing at the [build](https://github.com/getsentry/sentry/actions/runs/7806938986/job/21294303397?pr=64713).
@sentaur-athena If the build itself fails (not the diff), is that still non-blocking?

Reply:
yup if the build fails that is blocking!